wings/wallet/ethereum/web3_wallet_backend.py

```diff
@@ -0,0 +1,616 @@
+import asyncio
```

Comment:
Same here... we already have a copy in hummingbot right?

Reply:
fixed